### DISCUSSION TITLE
Is there a CHANGELOG, or some other means of finding out the current version number?

### DISCUSSION THREAD

**Discussion by unikitty37** [ueberdosis/tiptap#1285] [General Questions]
I was having a problem with v2.0.0-beta.22, and thought I should check to see if there was a new release. However, the only way I could find to see the current released version number was to let my IDE complete it in package.json, or to look on npmjs.com — and even then, I have no idea what the changes are!

Is there a changelog anywhere? If not, could there be one, please?

**Comment by philippkuehn** [comment]
For every package there is a `CHANGELOG.md`

Example: https://github.com/ueberdosis/tiptap/blob/main/packages/core/CHANGELOG.md

**Reply by unikitty37** [reply]
Ah, thanks — I was looking in the root 🤦
